messages/java/src/main/protobuf/gherkin.proto

```diff
@@ -0,0 +1,112 @@
+syntax = "proto3";
+import "sources.proto";
+package cucumber.messages;
```

Comment:
If this translates into a java package. Can we make it `io.cucumber.messages`?

Reply:
Will do!